sp/src/createallprojects.bat

```diff
@@ -1 +1,4 @@
-devtools\bin\vpc.exe /hl2 /episodic +everything /mksln everything.sln
+@echo off
+cls
+	devtools\bin\vpc.exe /hl2 /episodic +everything /mksln everything.sln
```

Comment:
What is the purpose of these batch file changes?

Reply:
oops this should not had to be here :sweat_smile: